playwright/test/concept.test.ts

```diff
@@ -27,206 +29,155 @@ const test = base.extend<{
+    test('has both works and image sections, each with about and using tabs', async ({
+      songsPage,
+    }) => {
+      // It has four tabs (two works, two images)
+      await expect(songsPage.worksAboutTab).toBeVisible();
+      await expect(songsPage.worksInTab).toBeVisible();
+      await expect(songsPage.imagesAboutTab).toBeVisible();
+      await expect(songsPage.imagesInTab).toBeVisible();
+      // The "works using" and "images using" panels should be visible initially
+      await expect(songsPage.worksInTabPanel).toBeVisible();
+      await expect(
+        songsPage.worksInTabPanel.getByRole('listitem')
+      ).not.toHaveCount(0);
+      await expect(songsPage.imagesInTabPanel).toBeVisible();
+      await expect(
+        songsPage.imagesInTabPanel.getByRole('listitem')
+      ).not.toHaveCount(0);
+
+      // It has links to filtered searches
+      await songsPage.worksInTab.click();
+      expect(await songsPage.allWorksLink.getAttribute('href')).toBe(
+        `/search/works?genres.label=${encodeURIComponent('"Songs"')}`
+      );
+
+      await songsPage.worksAboutTab.click();
+      expect(await songsPage.allWorksLink.getAttribute('href')).toBe(
+        `/search/works?subjects.label=${encodeURIComponent('"Songs"')}`
+      );
+
+      await songsPage.imagesInTab.click();
+      expect(await songsPage.allImagesLink.getAttribute('href')).toBe(
+        `/search/images?source.genres.label=${encodeURIComponent('"Songs"')}`
+      );
+
+      await songsPage.imagesAboutTab.click();
+      expect(await songsPage.allImagesLink.getAttribute('href')).toBe(
+        `/search/images?source.subjects.label=${encodeURIComponent('"Songs"')}`
+      );
+    });
   });
 
-  test('the "All images" link filters by the concept label', async ({
-    lithographsPage,
-  }) => {
-    expect(await lithographsPage.allImagesLink.getAttribute('href')).toBe(
-      `/search/images?source.genres.label=${encodeURIComponent(
-        '"Lithographs"'
-      )}`
-    );
+  test.describe('a Concept representing a Genre that is only used as a genre for both works and images', () => {
+    test('has both works and image sections showing records in that genre', async ({
+      lithographsPage,
+    }) => {
+      // Both images and works sections exist
+      await expect(lithographsPage.imagesHeader).toBeVisible();
+      await expect(lithographsPage.worksHeader).toBeVisible();
+      // There are no tabs, because there is only one group within each of the two sections
+      await expect(lithographsPage.worksAboutTab).not.toBeVisible();
+      await expect(lithographsPage.worksInTab).not.toBeVisible();
+      await expect(lithographsPage.imagesAboutTab).not.toBeVisible();
+      await expect(lithographsPage.imagesInTab).not.toBeVisible();
+
+      // It has links to filtered searches
+      expect(await lithographsPage.allWorksLink.getAttribute('href')).toBe(
```

Comment:
If we wanted to try an [auto-retrying assertion](https://playwright.dev/docs/test-assertions#auto-retrying-assertions) we could go with [.toHaveAttribute](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-have-attribute). .toBe works too, but [non-retrying assertions](https://playwright.dev/docs/test-assertions#non-retrying-assertions) have a higher chance of flakiness apparently!